source/features/comment-fields-keyboard-shortcuts.tsx

```diff
@@ -63,18 +63,14 @@ function init(): void {
 	onCommentFieldKeydown(eventHandler);
 }
 
-void features.add({
-	id: __filebasename,
-	description: 'Adds shortcuts to comment fields: `↑` to edit your previous comment; `esc` to blur field or cancel comment.',
-	screenshot: false,
-	shortcuts: {
-		'↑': 'Edit your last comment',
-		esc: 'Unfocuses comment field'
-	}
-}, {
+void features.add(__filebasename, {
 	include: [
 		pageDetect.hasRichTextEditor
 	],
 	awaitDomReady: false,
+	shortcuts: {
```

Comment:
Could the shortcuts be moved to on top of the include? I like it on top since it makes it stick out.

Was there a reason to move it here?

Reply:
Sure, I don't really care about the order in the loader object.